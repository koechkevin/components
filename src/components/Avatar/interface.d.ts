import { AvatarProps } from 'antd/lib/avatar';
import { CSSProperties, ReactNode } from 'react';

enum ChatStatus {
  online = 'online',
  offline = 'offline',
}

export interface Props extends AvatarProps {
  src?: string;
  dot?: boolean;
  status?: ChatStatus | string;
  size?: number | 'xsmall' | 'small' | 'medium' | 'large' | 'default';
  color?: string;
}
