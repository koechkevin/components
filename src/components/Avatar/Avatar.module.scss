@import '../../styles/global';

.avatar {
  &:global(.ant-avatar-square) {
    border-radius: 8px;
  }

  &:global(.ant-avatar img) {
    background-color: $color-basic-light-gray;
  }
}
