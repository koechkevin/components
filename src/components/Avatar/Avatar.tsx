import { Avatar as AntdAvatar } from 'antd';
import React, { FC } from 'react';

import styles from './Avatar.module.scss';

import { Props } from './interface.d';

export const Avatar: FC<Props> = (props) => {
  const { children, className, icon, src, shape, size, style, color } = props;
  const avatarSizes = {
    small: 32,
    medium: 40,
    large: 80,
    default: 'default',
  };
  const avatarSize = size && (avatarSizes[size] || size);
  const placeholderIcon = src ? icon : 'user';
  const classNames: string = [styles.avatar, className ? className : ''].join(' ');
  const avatarColor = color && !src ? { backgroundColor: color } : {};

  return (
    <AntdAvatar
      src={src}
      shape={shape}
      size={avatarSize}
      className={classNames}
      style={{ minWidth: avatarSize, ...avatarColor, ...style }}
      icon={!children ? placeholderIcon : ''}
    >
      {children}
    </AntdAvatar>
  );
};

Avatar.defaultProps = {
  size: 'default',
};

export default Avatar;
