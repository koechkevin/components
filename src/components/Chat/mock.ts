import { faCircle as faCircleOutline } from '@fortawesome/pro-regular-svg-icons';
import { faCircle } from '@fortawesome/pro-solid-svg-icons';

import { FileModel, MessageItemProps, SuggestionsUser } from '../../typings/Message';

export const file = {
  uid: '',
  size: 1024,
  type: 'pdf',
  name: 'Jane_Doe_Cv.pdf',
  lastModified: 1234,
  webkitRelativePath: '',
  lastModifiedDate: new Date(),
};

export const users: SuggestionsUser[] = [
  {
    id: '"6b550839-ac88-4691-a399-ae9d1465d073',
    name: 'Ke Yang',
    avatar: '',
    avatarColor: '#FB6D77',
    status: 'active',
    chatStatus: 'online',
    productId: 'company-portal',
    signature: 'UX / UI Designer',
    label: 'Label',
    icon: faCircle,
    iconColor: '#39c049',
  },
  {
    id: '02cbc571-6f03-4fea-85af-ba3527869090',
    name: 'Emery Muhozi',
    avatar: '',
    avatarColor: '#5D85CE',
    status: 'active',
    chatStatus: 'online',
    productId: 'company-portal',
    signature: undefined,
    icon: faCircleOutline,
    iconColor: '#959595',
  },
  {
    id: '662175e2-93a2-4977-99bd-89e66ca89a79',
    name: 'Chacha Zhou',
    avatar: '',
    avatarColor: '#46A094',
    status: 'active',
    chatStatus: 'offline',
    productId: 'candidate-portal',
    signature: 'Software Engineer',
    icon: faCircle,
    iconColor: '#39c049',
  },
  {
    id: '30d681e4-1551-49a5-9726-a745ad62838e',
    name: 'Morshed Alam',
    avatar: '',
    avatarColor: '#dfc475',
    status: 'active',
    chatStatus: 'offline',
    productId: 'candidate-portal',
    signature: 'Human Resources Manager',
    icon: faCircleOutline,
    iconColor: '#959595',
  },
  {
    id: '91d2fff3-ed29-47c7-bb21-3c478aff8caa',
    name: 'Alexander Fomin',
    avatar: '',
    avatarColor: '#6585c9',
    status: 'active',
    chatStatus: 'online',
    productId: 'candidate-portal',
    signature: 'Placement Manager',
    icon: faCircle,
    iconColor: '#39c049',
  },
  {
    id: '50915eca-0c64-4d53-aff7-545984f19af3',
    name: 'Sam Hash',
    avatar: '',
    avatarColor: '#666',
    status: 'active',
    chatStatus: 'online',
    productId: 'candidate-portal',
    signature: 'Placement Manager',
    icon: faCircleOutline,
    iconColor: '#959595',
  },
];

export const messages: MessageItemProps[] = [
  {
    id: '5d861e2775b4b200183e33a5',
    text:
      'HSLA color values are http://www.google.com an extension of HSL color values with an alpha channel - which specifies the opacity for a color. <br/>An HSLA color value is specified with: hsla(hue, saturation, lightness, alpha), where the alpha parameter defines the opacity. The alpha parameter is a number between 0.0 (fully transparent) and 1.0 (fully opaque).',
    forwarding: null,
    author: users[0],
    createdAt: '2019-10-22T16:40:29.240Z',
    updatedAt: '2019-10-22T16:40:29.240Z',
    threadInfo: {
      count: 4,
      authors: [users[1], users[2]],
      lastMessageAt: '2019-10-22T16:40:29.240Z',
    },
    isModified: true,
    file: {
      ...file,
      fileId: '1234',
      profileId: '',
    } as FileModel,
  },
  {
    id: '63762ebd-b714-4940-b989-b8db7c4002b4',
    text: 'A Version 4 UUID is a universally unique identifier that is generated using random numbers.',
    forwarding: null,
    author: users[0],
    createdAt: '2019-10-22T16:40:29.240Z',
    updatedAt: '2019-10-22T16:40:29.240Z',
    threadInfo: {
      count: 23,
      authors: [users[0], users[1], users[3], users[2]],
      lastMessageAt: '2019-10-22T16:40:29.240Z',
    },
    isModified: false,
  },
  {
    id: '63762ebd-b714-4940-b989-b8db7c4002b4',
    text: 'When a message has been deleted and had threaded messages in it',
    forwarding: null,
    author: users[0],
    createdAt: '2019-10-22T16:40:29.240Z',
    updatedAt: '2019-10-22T16:40:29.240Z',
    threadInfo: {
      count: 3,
      authors: [users[0]],
      lastMessageAt: '2019-10-22T16:40:29.240Z',
    },
    isModified: true,
    isDeleted: true,
  },

  {
    id: '63762ebd-b714-4940-b989-b8db7c4002b4',
    text: 'You are responsible for using \r\nthe UUIDs and assume any risk \r\nshowing multiple lines text content',
    forwarding: null,
    author: users[0],
    createdAt: '2019-10-22T16:40:29.240Z',
    updatedAt: '2019-10-22T16:40:29.240Z',
    threadInfo: {
      count: 23,
      authors: [users[0], users[1], users[4], users[2], users[3]],
      lastMessageAt: '2019-10-22T16:40:29.240Z',
    },
    isModified: false,
  },
  {
    id: '5d777045a2dcf20018ccad61',
    text: 'contain forwarding message below :) http://www.google.com',
    forwarding: {
      id: '5d6a1a38a2dcf20018ccad2d',
      threadId: '5d5e377a0ffa4f0018809fae',
      type: 'Normal',
      text:
        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed commodo, est nec tincidunt pharetra, dolor purus mattis sem, eget lobortis ex eros a arcu. Sed in turpis a nisl tincidunt faucibus.',
      createdAt: '2019-10-22T16:40:29.240Z',
      updatedAt: '2019-10-22T16:40:29.240Z',
      replies: [],
      author: users[1],
    },
    author: users[0],
    createdAt: '2019-10-22T16:40:29.240Z',
    updatedAt: '2019-10-22T16:40:29.240Z',
    threadInfo: {
      count: 23,
      authors: [users[0], users[1], users[3]],
      lastMessageAt: '2019-10-22T16:40:29.240Z',
    },
  },
  {
    id: '35cc2f98-1f55-4ecf-8f37-ecac8414290e',
    text:
      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam tempor dolor quis ex eleifend sollicitudin. Nullam eleifend, nisi et facilisis ullamcorper, justo quam cursus metus, tempus commodo nulla arcu eu nunc. Nullam pretium molestie maximus.',
    author: users[0],
    createdAt: '2019-10-22T16:40:29.240Z',
    updatedAt: '2019-10-22T16:40:29.240Z',
    threadInfo: null,
    file: {
      ...file,
      profileId: '',
      percent: 50,
      status: 'uploading',
    } as FileModel,
  },
];

export const texts = [
  {
    id: 'c5aadd40-6752-4d24-b27f-78aba29b0f71',
    text: `When mouse move entering, there's NO menu popped`,
    createdAt: '2019-10-22T16:40:29.240Z',
    updatedAt: '2019-10-22T16:40:29.240Z',
    menuShown: false,
    file: {
      ...file,
      fileId: 'abcd',
      profileId: '',
    } as FileModel,
  },
  {
    id: '3464f31c-fb4c-4835-9c3c-b6d3ef6f07e8',
    text: `Prethink gain traction low engagement nor vec, yet 4-blocker. We are running out of runway. Back to the drawing-board. Peel the onion agile let's put a pin in that.Can you send me an invite? I have zero cycles for this, but design thinking so you better eat a reality sandwich before you walk back in that boardroom, for five - year strategic plan.Viral engagement product market fit translating our vision of having a market leading platfrom.Deploy to production we need to have a Come to Jesus meeting with Phil about his attitude.Blue sky thinking.`,
    createdAt: '2019-10-22T16:40:29.240Z',
    updatedAt: '2019-10-22T16:40:29.240Z',
  },
  {
    id: 'b2c06e39-8f22-4294-a79a-3d921292543b',
    text: 'the textual message www.google.com \r\nthat has been changed',
    createdAt: '2019-10-22T16:40:29.240Z',
    updatedAt: '2019-10-22T16:40:29.240Z',
    isModified: true,
    menuShown: true,
  },
];
