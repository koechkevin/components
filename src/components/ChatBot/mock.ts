import { faCircle } from '@fortawesome/pro-solid-svg-icons';

import { BotMessage, Status } from '../../typings/Message.d';

const user = {
  id: '662175e2-93a2-4977-99bd-89e66ca89a79',
  name: 'Chacha Zhou',
  avatar: '',
  avatarColor: '#46A094',
  status: 'active',
  chatStatus: 'offline',
  productId: 'candidate-portal',
  signature: 'Software Engineer',
  icon: faCircle,
  iconColor: '#39c049',
};

export const botDirectMessage: BotMessage = {
  id: '5d861e2775b4b200183e33a5',
  type: Status.upcoming,
  isPrivate: true,
  createdAt: '2019-11-22T16:40:29.240Z',
  remindAt: '2019-12-8T16:40:29.240Z',
  updatedAt: '2019-10-22T16:40:29.240Z',
};

export const message = {
  id: '5d861e2775b4b200183e33a5',
  text:
    'HSLA color values are http://www.google.com an extension of HSL color values with an alpha channel - which specifies the opacity for a color. <br/>An HSLA color value is specified with: hsla(hue, saturation, lightness, alpha), where the alpha parameter defines the opacity. The alpha parameter is a number between 0.0 (fully transparent) and 1.0 (fully opaque).',
  forwarding: null,
  author: user,
  createdAt: '2019-10-22T16:40:29.240Z',
  updatedAt: '2019-10-22T16:40:29.240Z',
  threadInfo: {
    count: 4,
    authors: [user],
    lastMessageAt: '2019-10-22T16:40:29.240Z',
  },
  isModified: true,
};
