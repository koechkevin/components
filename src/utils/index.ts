import { BotAvatarType, ChatStatus, ErrorCodeMessage, FileExtension, ProductType, RolesConstants } from './constants';
import { getClientPos, getCroppedImg } from './imageCrop';
import timestampFormat, { dateFormat, shortTimeStampFormat } from './timestampFormat';
import {
  getDefaultCollapsedSubMenus,
  getFlatMenuKeys,
  getMenuMatches,
  getNameInitials,
  isEmpty,
  replacePath,
  resolveRole,
  selectIcon,
  urlConverter,
  urlToList,
} from './utils';

export {
  isEmpty,
  urlToList,
  dateFormat,
  replacePath,
  ChatStatus,
  BotAvatarType,
  ErrorCodeMessage,
  FileExtension,
  resolveRole,
  ProductType,
  RolesConstants,
  getMenuMatches,
  getFlatMenuKeys,
  getNameInitials,
  timestampFormat,
  shortTimeStampFormat,
  getDefaultCollapsedSubMenus,
  urlConverter,
  selectIcon,
  getCroppedImg,
  getClientPos,
};
