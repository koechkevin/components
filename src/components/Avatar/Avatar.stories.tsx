import { number, text } from '@storybook/addon-knobs';
import { storiesOf } from '@storybook/react';
import { Col, Row, Typography } from 'antd';
import React from 'react';

import { Center } from '../Center';
import { ListItem } from '../SideBar/CompanyModal';
import Avatar from './Avatar';

import logo from '../../icons/logo.svg';
import mockLogo from '../../icons/mocks/company.png';

const src = 'https://s3.amazonaws.com/uifaces/faces/twitter/bowbrick/128.jpg';
const size = number('size', 200);
const initials = text('initials', 'AA');
const { Text } = Typography;
const companyList: ListItem[] = [
  { companyId: '123', avatarUrl: logo, name: 'Airbus', count: 4, avatarColor: '#fb6d77' },
  { companyId: '1234', avatarUrl: mockLogo, name: 'Pepsi', count: 21, avatarColor: '#46a094' },
  { companyId: '12345', avatarUrl: '', name: 'Airbus', avatarColor: '#5d85ce' },
];

storiesOf('Avatar', module)
  .addDecorator((story) => <Center>{story()}</Center>)
  .add('default', () => <Avatar />, { notes: 'Avatar default styles' })
  .add(
    'with circle shape and image',
    () => (
      <Row>
        <Row gutter={36}>
          <Col span={8}>
            <Text strong>24</Text> xsmall{' '}
          </Col>
          <Col span={8}>
            <Text strong>40</Text> small{' '}
          </Col>
          <Col span={8}>
            <Text strong>80</Text> medium{' '}
          </Col>
        </Row>
        <Row gutter={36}>
          <Col span={8}>
            <Avatar src={src} size="small" />
          </Col>
          <Col span={8}>
            <Avatar src={src} size="medium" />
          </Col>
          <Col span={8}>
            <Avatar src={src} size="large" />
          </Col>
        </Row>
      </Row>
    ),
    {
      notes: 'Avatar with Circle shape and image',
    },
  )
  .add(
    'with square shape and image',
    () => (
      <Row>
        <Row gutter={36}>
          <Col span={8}>
            <Text strong>24</Text> xsmall
          </Col>
          <Col span={8}>
            <Text strong>40</Text> small
          </Col>
          <Col span={8}>
            <Text strong>80</Text> medium
          </Col>
        </Row>
        <Row gutter={36}>
          <Col span={8}>
            <Avatar src={src} shape="square" size="small" />
          </Col>
          <Col span={8}>
            <Avatar src={src} shape="square" size="medium" />
          </Col>
          <Col span={8}>
            <Avatar src={src} shape="square" size="large" />
          </Col>
        </Row>
      </Row>
    ),
    {
      notes: 'Avatar with square shape and image',
    },
  )
  .add(
    'with square shape and initials',
    () => (
      <Row>
        <Row>
          <Col span={12}>
            <Text strong>32</Text> small
          </Col>
          <Col span={12}>
            <Text strong>40</Text> large
          </Col>
        </Row>
        <Row gutter={36}>
          <Col span={12}>
            <Avatar size="small" shape="square" color="#46a094" style={{ fontWeight: 'bold', fontSize: 10 }}>
              {initials}
            </Avatar>
          </Col>
          <Col span={12}>
            <Avatar size="medium" shape="square" color="#8260da" style={{ fontWeight: 'bold', fontSize: 20 }}>
              {initials}
            </Avatar>
          </Col>
        </Row>
      </Row>
    ),
    {
      notes: 'Avatar with Square shape and initials',
    },
  )
  .add(
    'with circle shape and initials',
    () => (
      <Row>
        <Row gutter={36}>
          <Col span={8}>
            <Text strong>32</Text> small
          </Col>
          <Col span={8}>
            <Text strong>40</Text> medium
          </Col>
          <Col span={8}>
            <Text strong>80</Text> large
          </Col>
        </Row>
        <Row gutter={36}>
          <Col span={8}>
            <Avatar size="small" shape="circle" color="#46a094" style={{ fontWeight: 'bold', fontSize: 10 }}>
              {initials}
            </Avatar>
          </Col>
          <Col span={8}>
            <Avatar size="medium" shape="circle" color="#8260da" style={{ fontWeight: 'bold', fontSize: 20 }}>
              {initials}
            </Avatar>
          </Col>
          <Col span={8}>
            <Avatar size="large" shape="circle" color="#8260da" style={{ fontWeight: 'bold', fontSize: 20 }}>
              {initials}
            </Avatar>
          </Col>
        </Row>
      </Row>
    ),
    {
      notes: 'Avatar with Square shape and initials',
    },
  )
  .add('with custom size', () => <Avatar src={src} size={size} />, {
    notes: 'Avatar with custom size',
  })
  .add('with an icon', () => <Avatar icon="user" />, {
    notes: 'Avatar with an icon',
  })
  .add('company logo', () => (
    <Row gutter={32}>
      {companyList.map((company) => (
        <Col span={8} key={company.companyId}>
          <Avatar src={company.avatarUrl} size="medium" shape="square" color={company.avatarColor}>
            {company.name[0]}
          </Avatar>
        </Col>
      ))}
    </Row>
  ));
