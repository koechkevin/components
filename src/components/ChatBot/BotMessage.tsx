import { faEye } from '@fortawesome/pro-solid-svg-icons';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { List, Row } from 'antd';
import React, { FC, ReactNode, useState } from 'react';
import { Link } from 'react-router-dom';

import avatar from '../../icons/chat-bot-avatar/bot-avatar-default.png';
import { BotMessage as BotMessageTyped } from '../../typings/Message';
import { shortTimeStampFormat } from '../../utils';
import { Avatar } from '../Avatar';
import { Button } from '../Button';
import { Text } from '../Typography';
import styles from './Message.module.scss';

const { Item } = List;

interface Props {
  visible?: boolean;
  message: BotMessageTyped;
  poppedMenu?: ReactNode;
  onMark?: () => void;
  onSnooze?: () => void;
  onRemove?: () => void;
}

export const BotMessage: FC<Props> = (props) => {
  const { message, onMark, onSnooze, onRemove, poppedMenu } = props;
  const { id, createdAt, isPrivate } = message;
  const [visible, setVisible] = useState(false);

  return (
    <Item className={styles.botMessage} onMouseEnter={() => setVisible(true)} onMouseLeave={() => setVisible(false)}>
      {visible && id && poppedMenu}
      <Item.Meta
        avatar={
          <Avatar size={40} src={avatar}>
            Aida
          </Avatar>
        }
        title={
          <Row>
            <Row>
              <span className={styles.name}>Aida</span>
              <span className={styles.created}>{createdAt && shortTimeStampFormat(new Date(createdAt))}</span>
              {isPrivate && (
                <span style={{ color: '#0050c8', marginLeft: 12 }}>
                  <FontAwesomeIcon icon={faEye} /> Private : Only you can see this
                </span>
              )}
            </Row>
            <Row className={styles.text}>
              Got it! I'll remind you in <Text strong>20 minutes at 11.30 A.M today</Text> about{' '}
              <Link to={'/'}>this message</Link> from <Text strong>@JaneDoe</Text>
            </Row>
          </Row>
        }
        description={
          <Row className={styles.buttonGroup}>
            <Button ghost onClick={onMark}>
              Mark as Complete
            </Button>
            <Button ghost onClick={onSnooze}>
              Snooze
            </Button>
            <Button type="danger" ghost onClick={onRemove}>
              Delete
            </Button>
          </Row>
        }
      />
    </Item>
  );
};

export default BotMessage;
