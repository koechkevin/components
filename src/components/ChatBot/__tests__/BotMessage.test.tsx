import { BrowserRouter } from 'dva/router';
import { mount } from 'enzyme';
import React from 'react';

import { faEye } from '@fortawesome/pro-solid-svg-icons';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { shortTimeStampFormat } from '../../../utils';
import Avatar from '../../Avatar/Avatar';
import BotMessage from '../BotMessage';
import { botDirectMessage } from '../mock';

describe('Tests for the BotMessage component', () => {
  const props = {
    poppedMenu: <div>Mock Node</div>,
    message: botDirectMessage,
    onMark: jest.fn(),
    onSnooze: jest.fn(),
    onRemove: jest.fn(),
  };

  const wrapper = mount(
    <BrowserRouter>
      <div>
        <BotMessage {...props} />
      </div>
    </BrowserRouter>,
  );

  const icon = <FontAwesomeIcon icon={faEye} />;

  it('should render Bot Message without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render the bot avatar', () => {
    const avatar = wrapper.find(Avatar).first();
    expect(avatar.props().size).toEqual(40);
  });

  it('should render correct timestamp for bot message', () => {
    expect(wrapper.contains(shortTimeStampFormat(new Date(props.message.createdAt)))).toEqual(true);
  });

  it('should render Private message icon', () => {
    expect(wrapper.contains(icon)).toEqual(true);
  });

  it('should render respective buttons for chat bot messages', () => {
    expect(wrapper.contains('Mark as Complete')).toEqual(true);
    expect(wrapper.contains('Snooze')).toEqual(true);
    expect(wrapper.contains('Delete')).toEqual(true);
  });

  it('should simulate clicking the mark as read button', () => {
    const markCompleteBtn = wrapper.findWhere((n) => n.text() === 'Mark as Complete').first();
    markCompleteBtn.simulate('click');

    expect(props.onMark).toHaveBeenCalled();
  });

  it('should simulate clicking the snooze button', () => {
    const markCompleteBtn = wrapper.findWhere((n) => n.text() === 'Snooze').first();
    markCompleteBtn.simulate('click');

    expect(props.onSnooze).toHaveBeenCalled();
  });

  it('should simulate clicking the delete button', () => {
    const markCompleteBtn = wrapper.findWhere((n) => n.text() === 'Delete').first();
    markCompleteBtn.simulate('click');

    expect(props.onRemove).toHaveBeenCalled();
  });
});
