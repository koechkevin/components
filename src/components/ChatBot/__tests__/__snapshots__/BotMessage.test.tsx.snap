// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for the BotMessage component should render Bot Message without crashing 1`] = `ReactWrapper {}`;
